packages/core/src/sync/sync-api.ts

```diff
@@ -186,21 +198,37 @@ export class SyncApi implements ISyncApi {
    * Also keeps in sync with new anchors.
    */
   async startModelSync(
-    startBlock: number,
-    endBlock: number,
-    models: string | string[]
+    models: StreamID | StreamID[],
+    startBlock = INITIAL_INDEXING_BLOCK,
+    endBlock?
   ): Promise<void> {
-    const modelIds = Array.isArray(models) ? models : [models]
+    if (!this.syncConfig.on) return
```

Comment:
Maybe this should throw an error considering this method shouldn't be called in the first place if sync is off, no?

Reply:
This function is being called from the AdminApi where it has no concept if sync should be on or not, similar to how local index api is used. 

Adding a model to the index currently does not throw even if  `process.env.CERAMIC_ENABLE_EXPERIMENTAL_COMPOSE_DB != 'true'`. I am maintaining that consistency for now.

I am not sure what an admin api user is going to expect or what the cli(?) user would expect.

Reply:
OK thanks, indeed if the method callers don't do these checks better have this check here.